src/middleware.rs

```diff
@@ -305,12 +330,38 @@ pub trait MainPod: fmt::Debug + DynClone {
 // impl Clone for Box<dyn SignedPod>
 dyn_clone::clone_trait_object!(MainPod);
 
+/// This is a filler type that fulfills the MainPod trait and always verifies.  It's empty.  This
+/// can be used to simulate padding in a circuit.
+#[derive(Debug, Clone)]
+pub struct NoneMainPod {}
+
+impl MainPod for NoneMainPod {
+    fn verify(&self) -> bool {
+        true
+    }
+    fn id(&self) -> PodId {
+        PodId(NULL)
+    }
+    fn pub_statements(&self) -> Vec<Statement> {
+        Vec::new()
+    }
+    fn into_any(self: Box<Self>) -> Box<dyn Any> {
+        self
+    }
+}
+
+// TODO: Figure out a way to signal which signed_pods entries and which main_pods statements need
+// to be made public.  Idea: introduce an operation called reveal, which the backend translates to
```

Comment:
@arnaucube this is resolved with the `public_statements` field, I forgot to remove this comment.

Reply:
thanks, removed the comment at https://github.com/0xPARC/pod2/pull/35